content/en/docs/started/installing-kubeflow.md

```diff
@@ -0,0 +1,118 @@
+To read more about the components and architecture of Kubeflow, please see the <a href="/docs/started/kubeflow-overview/">Kubeflow Overview</a> page.
+
+There are two pathways to get up and running with Kubeflow, you may either:
+1. Use a [packaged distribution](#packaged-distributions)
+1. Use the [manifests](#manifests) (advanced)
+
+<a id="packaged-distributions"></a>
+## Packaged distributions
+
+{{% alert color="warning" %}}
+Packaged distributions are developed and supported by their respective maintainers, the Kubeflow community does not currently endorse or certify any distribution.
+{{% /alert %}}
+
+<b>See the table below for a list of options and links to documentation:</b>
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Name</th>
+        <th>Maintainer</th>
+        <th>Platform</th>
+        <th>Link - Docs</th>
+        <th>Link - External Website</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Kubeflow on AWS</td>
+        <td>AWS</td>
+        <td>Amazon Elastic Kubernetes Service (EKS)</td>
+        <td><a href="/docs/distributions/aws/">Docs</a></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Kubeflow on Azure</td>
+        <td>Azure</td>
+        <td>Azure Kubernetes Service (AKS)</td>
+        <td><a href="/docs/distributions/azure/">Docs</a></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Kubeflow on GCP</td>
+        <td>Google Cloud</td>
+        <td>Google Kubernetes Engine (GKE)</td>
+        <td><a href="/docs/distributions/gke/">Docs</a></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Kubeflow on IBM Cloud</td>
+        <td>IBM Cloud</td>
+        <td>IBM Cloud Kubernetes Service (IKS) </td>
+        <td><a href="/docs/distributions/ibm/">Docs</a></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Kubeflow on Openshift</td>
+        <td>IBM Cloud</td>
+        <td>Openshift</td>
+        <td><a href="/docs/distributions/openshift/">Docs</a></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Argoflow</td>
+        <td>Argoflow Community</td>
+        <td>Conformant Kubernetes</td>
+        <td>N/A</td>
+        <td><a href="https://github.com/argoflow/argoflow">External Website</a></td>
+      </tr>
+      <tr>
+        <td>Charmed Kubeflow</td>
+        <td>Canonical</td>
+        <td>Conformant Kubernetes</td>
+        <td><a href="/docs/distributions/charmed/">Docs</a></td>
+        <td><a href="https://charmed-kubeflow.io/docs">External Website</a></td>
+      </tr>
+      <tr>
+        <td>MicroK8s Kubeflow add-on</td>
+        <td>Canonical</td>
+        <td>MicroK8S</td>
+        <td><a href="/docs/distributions/microk8s/">Docs</a></td>
+        <td><a href="https://microk8s.io/docs/addon-kubeflow">External Website</a></td>
+      </tr>
+      <tr>
+        <td>MiniKF</td>
```

Comment:
Arrikto MiniKF

Reply:
I am fine to change it, but is that the "name" of the distribution?

Reply:
@shannonbradshaw I am fine to change it, but is that the "name" of the distribution?

Reply:
I have made this change, feel free to change it later.